template/src/main.js

```diff
@@ -0,0 +1,16 @@
+import Vue from 'vue';
+import App from './App.vue';
+import singleSpaVue from 'single-spa-vue';
+
+Vue.config.productionTip = false;
+
+const vueLifecycles = singleSpaVue({
+  Vue,
+  appOptions: {
+    render: (h<% if(isTs) {%>: any<% } %>) => h(App),
```

Comment:
This is an [ejs](https://cli.vuejs.org/dev-guide/plugin-dev.html#creating-new-templates) template. We need the `: any` for Typescript linters not to freak out about us having an "implict any return"